add helm chart and client's review is done (#1)
LucaRocco/liqo_resources_dashboard: chart/templates/NOTES.txt

> {{ .Chart.Name }} {{ .Chart.Version }} installed and the release is called {{ .Release.Name }}

Reviewer: Would personally either drop or enrich (such as you can access the dashboard at ...), otherwise it is of little value.

{{ .Chart.Name }} {{ .Chart.Version }} installed and the release is called {{ .Release.Name }}.

> All the resources have been deployed correctly in the {{ if not .Release.namespace }}{{ print "default" }}{{ else }}{{ print "%s" .Release.namespace }}{{ end }} namespace

Reviewer: ```suggestion
All the resources have been deployed correctly in the {{ print "%s" (default .Release.namespace "default") }} namespace.
```

Something like this should work, but it shall be tested.

> The dashboard is available at {{ if .Values.tls.secretName }}{{ print "https://" }}{{ else }}{{ print "http://" }}{{ end }}{{ if .Values.host }}{{ .Values.host }}{{ else }}{{ print "localhost" }}{{ end }}

Reviewer: ```suggestion
The dashboard is available at {{ if .Values.tls.secretName }}{{ print "https://" }}{{ else }}{{ print "http://" }}{{ end }}{{ if .Values.host }}{{ .Values.host }}{{ else }}{{ print "localhost" }}{{ end }}.
```

> Remember that the dashboard works if and only if liqo is installed on the cluster

Reviewer: ```suggestion
Remember that this dashboard requires liqo to be installed in the cluster to display useful information.
```
